docs/dg/settingUp.md

```diff
@@ -10,7 +10,7 @@
 
 **Prerequisites:**
 * **JDK `1.8.0_60`** up to **`17`** ([download :fas-download:](https://www.oracle.com/technetwork/java/javase/downloads/index.html)).
-* **Node.js** `14.19.0` to the latest minor version for `16` ([download :fas-download:](https://www.npmjs.com/get-npm)).
+* **Node.js** `16.19.1` to the latest minor version for `18` ([download :fas-download:](https://www.npmjs.com/get-npm)).
```

Comment:
Has Node 18 been proven to work with RepoSense? Mentioning it here implies that we are sure it does.

Reply:
I have tried the Gradle tasks and CI with both Node 16 and 18, which work for me. There are some warnings about deprecated dependencies though:

<img width="1003" alt="Screenshot 2023-03-25 at 1 30 02 PM" src="https://user-images.githubusercontent.com/68203159/227698301-b620de25-f856-4fbd-8bca-829baf8dc5d0.png">

Reply:
These deprecated dependencies shouldn't be a problem for now. 

Some of them might be resolved by updating some of the dependency version numbers in `package.json`, but this can be done in a separate PR.